src/core/server/http/router/router.ts

```diff
@@ -21,28 +21,45 @@ import { ObjectType, TypeOf, Type } from '@kbn/config-schema';
 import { Request, ResponseObject, ResponseToolkit } from 'hapi';
 import Boom from 'boom';
 
+import { Stream } from 'stream';
 import { Logger } from '../../logging';
 import { KibanaRequest } from './request';
 import { KibanaResponseFactory, kibanaResponseFactory, IKibanaResponse } from './response';
-import { RouteConfig, RouteConfigOptions, RouteMethod, RouteSchemas } from './route';
+import {
+  RouteConfig,
+  RouteConfigOptions,
+  RouteMethod,
+  RouteSchemas,
+  validBodyOutput,
+} from './route';
 import { HapiResponseAdapter } from './response_adapter';
 import { RequestHandlerContext } from '../../../server';
 import { wrapErrors } from './error_wrapper';
 
-interface RouterRoute {
+/**
+ * Definition of a route
+ *
+ * @public
+ */
+export interface RouterRoute {
```

Comment:
oops that was an internal interface. Now `Boom` is leaking as a valid `handler` outcome, but `handler` is an internal concept

Reply:
You are right! Moved it back to internal